exporters/otlp/CMakeLists.txt

```diff
@@ -36,7 +36,8 @@ if(WITH_OTLP_GRPC)
 
   target_link_libraries(
     opentelemetry_exporter_otlp_grpc_client
-    PUBLIC opentelemetry_sdk opentelemetry_ext opentelemetry_proto)
+    PUBLIC opentelemetry_sdk opentelemetry_common opentelemetry_ext
```

Comment:
This problem is already fixed in #2097 .

Reply:
Please correct me otherwise, but I don't see this is fixed with #2097. Also validated by building shared libs from that PR, and below is the result:
```
$ ldd -r -d libopentelemetry_*so  2>&1 | grep undefin
undefined symbol: _ZNK13opentelemetry2v13sdk8resource8Resource13GetAttributesEv (./libopentelemetry_exporter_ostream_metrics.so)
undefined symbol: _ZN13opentelemetry2v13sdk6common12internal_log16GlobalLogHandler18GetHandlerAndLevelEv        (./libopentelemetry_exporter_otlp_grpc_client.so)
undefined symbol: _ZN13opentelemetry2v13sdk6common12internal_log16GlobalLogHandler18GetHandlerAndLevelEv        (./libopentelemetry_http_client_curl.so)
undefined symbol: _ZN13opentelemetry2v13sdk6common12internal_log16GlobalLogHandler18GetHandlerAndLevelEv        (./libopentelemetry_exporter_otlp_http_client.so)
undefined symbol: _ZN13opentelemetry2v13sdk6common12internal_log16GlobalLogHandler18GetHandlerAndLevelEv        (./libopentelemetry_http_client_curl.so)
undefined symbol: _ZN4grpc24g_core_codegen_interfaceE   (./libopentelemetry_proto_grpc.so)
undefined symbol: _ZN4grpc6g_glipE      (./libopentelemetry_proto_grpc.so)
undefined symbol: _ZN4absl12lts_202103245MutexD1Ev      (./libopentelemetry_proto_grpc.so)
```

Reply:
Sorry, I mean the proto dependency of opentelemetry_exporter_otlp_grpc_client is added in #2097 . I didn't see  opentelemetry_common before and it's still useful.
I can solve the conflict later.